### PR TITLE
Docs: add rust environment setup for kata 3.0

### PR DESCRIPTION
**Pull request by Tim-0731-Hzt**
add more details for rust set up in kata 3.0 install guide

Fixes: #4720
Signed-off-by: Zhongtao Hu <zhongtaohu.tim@linux.alibaba.com>

### CODE REVIEW

**Comment by studychao** [comment]
Could you add the reminder that rust version 1.58 is needed?

**Comment by Tim-0731-Hzt** [comment]
> Could you add the reminder that rust version 1.58 is needed?

fixed

**Comment by studychao** [comment]
Maybe some insturction about how to switch to rust 1.58 after rust installation?

**Comment by Tim-0731-Hzt** [comment]
> Maybe some insturction about how to switch to rust 1.58 after rust installation?

Fixed

File: docs/install/kata-containers-3.0-rust-runtime-installation-guide.md [hunk 1]
```diff
@@ -44,6 +44,35 @@ architectures:
...
 
+### Rust Environment Set Up
+
+* Download `Rustup` and install  `Rust`
+    > **Notes:**
+    > Rust version 1.58 is needed
+    
+    ```
+    $ curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh
+    $ rustup install 1.58
```
[the remark this answers is not in the document]
Comment (Tim-0731-Hzt) [reply docs/install/kata-containers-3.0-rust-runtime-installation-guide.md:57]: Thanks. it's been fixed

**Review by liubin — APPROVED** [review]

**Review by bergwolf — APPROVED** [review]

**Pull request merged**